CFC_DataCollector/trip_analysis/kmedoid.py

```diff
@@ -0,0 +1,122 @@
+import random
+import sys
+
+#code based on K_medoid_2.py in CFC_WebApp/main
```

Comment:
If so, can you delete the one in main? It really shouldn't be in main because it is not really part of the webapp.